packages/agoric-cli/README.md

```diff
@@ -2,37 +2,31 @@
 
 ## Prerequisites
 
-### Vagrant
+### Use as part of Agoric SDK
 
-To run a standardized Linux distribution with all the required development tools, you probably want [Vagrant](https://www.vagrantup.com/docs/):
+If you have cloned the Agoric SDK, you can use the Agoric CLI directly.
 
 ```sh
```

Comment:
The user should have made sure to do `yarn install` and `yarn build` from workspace root, right? Could we mention that here? I'd like to make the instructions fool-proof for my future self :)

Reply:
I would prefer to say that the user needs to follow the instructions for installing the Agoric SDK, so I'll do that.

Reply:
ok, sounds good